tests/integration/Makefile

```diff
@@ -28,59 +28,92 @@ ifeq ($(S2N_CORKED_IO),true)
 	HANDSHAKE_TEST_PARAMS:=--use_corked_io $(HANDSHAKE_TEST_PARAMS)
 endif
 
+#Make sure these path variables have something.
 ifeq ($(OPENSSL_0_9_8_INSTALL_DIR),)
 	OPENSSL_0_9_8_INSTALL_DIR:=${CURDIR}/../../test-deps/openssl-0.9.8
 endif
+ifeq ($(OPENSSL_1_1_1_INSTALL_DIR),)
+	OPENSSL_1_1_1_INSTALL_DIR:=${CURDIR}/../../test-deps/openssl-1.1.1
+endif
+ifeq ($(GNUTLS_INSTALL_DIR),)
+	GNUTLS_INSTALL_DIR:=${CURDIR}/../../test-deps/gnutls
+endif
+ifeq ($(OQS_OPENSSL_1_1_1_INSTALL_DIR),)
+	OQS_OPENSSL_1_1_1_INSTALL_DIR:=${CURDIR}/../../test-deps/oqs_openssl-1.1.1
+endif
 
-TESTS = tls13 client_endpoints dynamic_record old_s_client s_client s_server gnutls_cli gnutls_serv
+TESTS = tls13 client_endpoints dynamic_record old_s_client s_client s_server pq_handshake
 
-.PHONY : all
 ifndef S2N_NO_PQ
-TESTS += pq_handshake
+	TESTS += pq_handshake
 endif
```

Comment:
It looks like you're always running the pq_handshake test, but running it twice if S2N_NO_PQ isn't set. Am I missing something here? Have you tested with S2N_NO_PQ?